### DISCUSSION TITLE
get column info when onChange update

### DISCUSSION THREAD

**Discussion by railty** [nick-keller/react-datasheet-grid#136] [Q&A] [answered]
is it possible to get column info when onChange update? ie, can you pass a fromColumnIndex, toColumnIndex in the operations, only needed for update opertion?

This will help if user want to update the server data, we need know the column in the sql

**Comment by nick-keller** [comment, marked as answer]
Hello @railty ,
You might want to look at [this example](https://react-datasheet-grid.netlify.app/docs/examples/tracking-rows-changes) first.
Usually, you do not need to know which columns were updated, just update the entire row. And if you really need to know, you can compare the row to the previous version of the row.

The reason this information is not available is that a column can update an entire row, not necessarily just a single key. And multiple columns could update the same key, so their is no 1-1 relationship between columns and key.
